anta/tests/routing/bgp.py

```diff
@@ -928,19 +928,16 @@ class VerifyBGPAdvCommunities(AntaTest):
     This test performs the following checks for each specified peer:
 
       1. Verifies that the peer is found in its VRF in the BGP configuration.
-      2. Validates that all required community types are advertised:
-        - Standard communities
-        - Extended communities
-        - Large communities
+      2. Validates that given community types are advertised. If not provided, validates all the advertised communities.
```

Comment:
```suggestion
      2. Validates that given community types are advertised. If not provided, validates that all communities (standard, extended, large) are advertised.
```
We should also update the test docstring: `Verifies the advertised communities for BGP IPv4 peer(s).`

Reply:
Updated